lib/twirp/rails/logging/adapter.rb

```diff
@@ -6,19 +6,20 @@ module Twirp
   module Rails
     module Logging
       module Adapter# :nodoc:
+
         def self.install
           return unless defined?(ActiveSupport::Notifications)
 
-          Twirp::Rails::Routes.on_create_service do |service|
-            Logging::Adapter.instrument service
+          Twirp::Rails::Routes.on_create_service do |service_wrapper|
+            Logging::Adapter.instrument service_wrapper
           end
         end
 
-        def self.instrument(service)
+        def self.instrument(service_wrapper)
```

Comment:
⛏️ this method is doing 5 conceptually related but different things.

I feel like this should just be calling out to individual methods that handle those things.  Top of my head name proposals:

* configure_before_route_request
* configure_before
* configure_on_error
* configure_on_success
* configure_exception_raised

Reply:
@masukomi not sure what you mean are you referring to drying this up to something like:

```
def self.instrument(service_wrapper)
  configure_before_route_request(service_wrapper)
  ...
end
```

Reply:
yup. exactly. each one of those block in it feels like it should be its own method to me.